handler/par/flow_pushed_authorize.go

```diff
@@ -0,0 +1,88 @@
+package par
```

Comment:
Does it make sense to introduce a handler for this? To me, it appears as if there would be only one implementation of PAR? Or are there already multiple ones?

Reply:
The intent is to allow for additional handlers to extend the spec. For example - there can be FAPI-specific constraints that an implementer might want to implement. So while the basic implementation is just one handler, it makes sense for a SDK like Fosite to offer extension points. This is something I appreciate with the implementation for the TokenEndpointHandler and for the AuthorizeEndpointHandler, so I would like to preserve that extensibility.

Reply:
Ok, that makes sense to me!